ethpm_cli/install.py

```diff
@@ -47,11 +54,39 @@ def install_package(pkg: Package, config: Config) -> None:
     tmp_pkg_dir = Path(tempfile.mkdtemp())
     write_pkg_installation_files(pkg, tmp_pkg_dir, config.ipfs_backend)
 
-    # Copy temp pacakge directory to ethpm dir namespace
+    # Copy temp package directory to ethpm dir namespace
     dest_pkg_dir = config.ethpm_dir / pkg.alias
     validate_parent_directory(config.ethpm_dir, dest_pkg_dir)
     shutil.copytree(tmp_pkg_dir, dest_pkg_dir)
-    update_ethpm_lock(pkg, (config.ethpm_dir / "ethpm.lock"))
+    install_to_ethpm_lock(pkg, (config.ethpm_dir / "ethpm.lock"))
+
+
+def list_installed_packages(config: Config) -> None:
+    for pkg_data in get_installed_packages(config.ethpm_dir):
+        logger.info(pkg_data)
+
+
+@to_list
+def get_installed_packages(ethpm_dir: Path) -> Iterable[str]:
+    installed_pkgs = reversed(sorted(ethpm_dir.glob("**/manifest.json")))
```

Comment:
I don't think this is a safe mechanism to use.  A package could maliciously ship a file called `manifest.json` which would be caught by this mechanism and corrupt the displayed data.